### PR TITLE
update to melos 3.0.0

### LINKED ISSUE #244: Update to melos 3.0.0
melos v3.0.0 broke our current github action setup.
We need to create a root-`pubspec.yaml` specifying the melos version.
Also we need to fix our GitHub Actions setup.

### PR DESCRIPTION
**Pull request by DasProffi**
closes #244

### CODE REVIEW

**Comment by use-to** [comment]
[`be6bda8` (#242)](https://github.com/helpwave/mobile-app/pull/242/commits/be6bda84f073b906265e0db55fe3652622cad238)

**Comment by DasProffi** [comment]
> [`be6bda8` (#242)](https://github.com/helpwave/mobile-app/pull/242/commits/be6bda84f073b906265e0db55fe3652622cad238)

We should upgrade instead of just fixing the problem, shouldn't we?

**Comment by use-to** [comment]
> > [`be6bda8` (#242)](https://github.com/helpwave/mobile-app/pull/242/commits/be6bda84f073b906265e0db55fe3652622cad238)
> 
> We should upgrade instead of just fixing the problem, shouldn't we?

Yeah obviously we should upgrade the version.
BUT we can apply this simple fix in order to merge #241 and #242 .

The reason why I referenced that commit is that we should lock the version to `3.0.0` to avoid these problems in the future.

File: .gitignore [hunk 1]
```diff
@@ -12,8 +12,7 @@ build/
 coverage/
 lib/generated_plugin_registrant.dart
 # For library packages, don’t commit the pubspec.lock file.
-# Regenerating the pubspec.lock file lets you test your package against the latest compatible versions of its dependencies.
-# See https://dart.dev/guides/libraries/private-files#pubspeclock
+# exception here because of melos see https://melos.invertase.dev/guides/migrations#200-to-300
 pubspec.lock
```
Comment (use-to) [inline .gitignore:16]:
```suggestion
```

**Review by use-to — APPROVED** [review]

**Pull request merged**